source/vibe/core/drivers/libasync.d

```diff
@@ -117,6 +119,11 @@ final class LibasyncDriver : EventDriver {
 		logTrace("Deleting event driver");
 		m_break = true;
 		getEventLoop().exit();
+		s_refCount.atomicOp!"-="(1);
+		if (s_refCount.atomicLoad() == 0) {
```

Comment:
Hm I'm probably not thinking things through enough at this point. I just can't wait to mark it done :)